### PR TITLE
Update Next.js example

### PR DESCRIPTION
**Pull request by willross**
https://github.com/hjylewis/babel-plugin-inline-i18n-messages/issues/6

### CODE REVIEW

File: examples/nextjs-example/README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 1. `npm install`
-1. `npm run dev`
+2. `npm run dev` OR `npm run build && npm run start`
+
+#### NOTE:
+
+Next.js doesn't support exporting multiple webpack configs in `next.config.js` (https://github.com/zeit/next.js/issues/1306), so each page will have relevant messages for **all** locales bundled in. This leads to increased bundle size as compared to the vanilla example.
+
+More: https://github.com/hjylewis/babel-plugin-inline-i18n-messages/issues/6
```
Comment (hjylewis) [inline examples/nextjs-example/README.md:8]: Love it 🎉

File: examples/nextjs-example/.babelrc [hunk 2]
```diff
@@ -1,19 +1,3 @@
...
-        "addMessagesSource": "i18n/messages"
-      }
-    ],
-    ["module-resolver", {
-      "alias": {
-        "i18n": "./pages/i18n"
-      }
-    }]
-  ]
+  "presets": ["next/babel"]
```
Comment (hjylewis) [inline examples/nextjs-example/.babelrc:2]: I think you can delete this file

File: examples/nextjs-example/next.config.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      use: [
+        options.defaultLoaders.babel,
+        {
+          loader: "babel-loader",
+          options: {
+            plugins: [
+              [
+                "inline-i18n-messages",
+                {
+                  locale: "en",
```
Comment (hjylewis) [inline examples/nextjs-example/next.config.js:20]:
```suggestion

```

File: examples/nextjs-example/.babelrc (hunk 2 above)
Comment (willross) [reply examples/nextjs-example/.babelrc:2]: Deleting it breaks everything for me 👀

File: examples/nextjs-example/README.md (hunk 1 above)
Comment (willross) [reply examples/nextjs-example/README.md:8]: 🙏

File: examples/nextjs-example/.babelrc (hunk 2 above)
Comment (hjylewis) [reply examples/nextjs-example/.babelrc:2]: 👀 no worry then

**Pull request merged**